runner/client/run_pod.py

```diff
@@ -16,17 +27,24 @@ def __init__(self, model):
         self.api = None
 
     def __enter__(self):
-        ow = OpenWeights()
-        deployment_job = ow.deployments.create(
-            model=self.model,
-            max_model_len=4096
-        )
-        self.api = TemporaryApi(ow, deployment_job['id'], client_type=OpenAI)
-        self.client = self.api.up()
-        return self.client
+        if self.model in DEPLOYMENTS:
+            self.client = DEPLOYMENTS[self.model]
+            return self.client
+        else:
+            raise ValueError('Did not find deployment for', self.model)
+        # ow = OpenWeights()
```

Comment:
TBH I don't understand how that's supposed to work. We never call register_deployment, right? Because most of the stuff is commented?